### PR TITLE
Subjects

### PR DESCRIPTION
**Pull request by nevizir**

### CODE REVIEW

File: create_db.sql [hunk 1]
```diff
@@ -3,3 +3,10 @@
 create database TestDatabase;
 
 use TestDatabase;
+
+
+create table Subjects 
+(
+	Id int primary key,
```
Comment (Pasechnyk) [inline create_db.sql]: Add identity to key

**Review by Pasechnyk — CHANGES_REQUESTED** [review]

File: create_db.sql [hunk 2]
```diff
@@ -3,3 +3,10 @@
 create database TestDatabase;
 
 use TestDatabase;
+
+
+create table Subjects
```
Comment (vladtymo) [inline create_db.sql]: You should add the IDENTITY property to the primary key

File: create_db.sql [hunk 3]
```diff
@@ -3,3 +3,10 @@
 create database TestDatabase;
 
 use TestDatabase;
+
+
+create table Subjects 
+(
+	Id int primary key,
+	Name nvarchar(100) not null,
```
Comment (vladtymo) [inline create_db.sql]: Must be unique and not null

**Review by vladtymo — CHANGES_REQUESTED** [review]

**Comment by vladtymo** [comment]
You should add the insert commands also

**Review by vladtymo — APPROVED** [review]

**Review by Pasechnyk — APPROVED** [review]

**Pull request merged**